schema/gene.js

```diff
@@ -17,6 +16,8 @@ import {
 
 const GeneType = new GraphQLObjectType({
   name: 'Gene',
+  interfaces: [NodeInterface],
+  isTypeOf: (obj) => _.has(obj, 'name'),
```

Comment:
Just a name does not uniquely identify a Gene. Be sure to use such fields instead and that they are included in Gravity’s ‘short’ JSON payloads.